packages/phoenix-event-display/src/loaders/jivexml-loader.ts

```diff
@@ -633,29 +633,25 @@ export class JiveXMLLoader extends PhoenixLoader {
   public getJets(firstEvent: Element, eventData: { Jets: any }) {
     const jetsHTML = firstEvent.getElementsByTagName('Jet');
     const jetCollections = Array.from(jetsHTML);
-    const nameOfCollection = 'AntiKt4TopoJets';
     for (const jetColl of jetCollections) {
-      // Extract the only collection we (currently) care about
-      // if (jetColl.getAttribute("storeGateKey")==nameOfCollection){
       const numOfJets = Number(jetColl.getAttribute('count'));
 
       // The nodes are big strings of numbers, and contain carriage returns. So need to strip all of this, make to array of strings,
       // then convert to array of numbers
       const phi = this.getNumberArrayFromHTML(jetColl, 'phi');
       const eta = this.getNumberArrayFromHTML(jetColl, 'eta');
       const energy = this.getNumberArrayFromHTML(jetColl, 'energy');
-
+      const coneR = this.getNumberArrayFromHTML(jetColl, 'coneR');
       const temp = []; // Ugh
       for (let i = 0; i < numOfJets; i++) {
         temp.push({
-          coneR: 0.4,
+          coneR: coneR[i] ? coneR[i] : 0.4, // Set default of 0.4, since some JiveXML files might not have this.
```

Comment:
There is the useful "nullish coalescing" operator for this. So `coneR[i] ? coneR[i] : 0.4` will become:

```ts
coneR[i] ?? 0.4
```

Reply:
Ah right - I have seen that before, but there's so many shortcuts with java/type script that my poor C++ brain struggles to keep up! ;-)